eclipse-scout-core/src/table/columns/Column.ts

```diff
@@ -161,6 +161,7 @@ export class Column<TValue = string> extends PropertyEventEmitter implements Col
     icons.resolveIconProperty(this, 'headerIconId');
     this._setTable(this.table);
     this._setAutoOptimizeWidth(this.autoOptimizeWidth);
+    this.sortActive = this.sortIndex >= 0;
```

Comment:
sortActive can be passed by the model -> set this.sortActive to null in the constructor and use scout.nvl(this.sortActive, this.sortIndex >= 0) here. Please also adjust the jsdoc in ColumnModel.ts for sortActive to something like this: Default is true if {@link sortIndex} is >= 0, otherwise it is false.